master/getting-started/kubernetes/tutorials/advanced-policy.md

```diff
@@ -194,6 +194,8 @@ metadata:
 spec:
   selector: calico/k8s_ns == 'advanced-policy-demo'
   order: 500
+  types:      # supported by Calico v2.6+ only
+  - egress
```

Comment:
I don't see the benefit of adding these lines to this example, as it makes the example look more complex, but does not actually change the behaviour *at all*.

(In other words, types 'egress' would be the default for this policy anyway, given that only egress rules are specified; hence there is no benefit from specifying types 'egress' explicitly.)